test/testtokenize.cpp

```diff
@@ -7365,7 +7366,7 @@ class TestTokenizer : public TestFixture {
         ASSERT_EQUALS("a ? ( b < c ) : d > e", tokenizeAndStringify("a ? b < c : d > e"));
     }
 
-    std::string testAst(const char code[],bool verbose=false) {
+    std::string testAst(const char code[], bool verbose = false, bool z3 = false) {
```

Comment:
I'm not at all happy with this signature. I think it would make more sense to accept an enum rather than the two bools. I haven't done this at this stage as it would require changing all existing use cases which would pollute the PR.

Reply:
yes I totally agree it would be nicer with a enum but let's do that a bit later.